src/main/webapp/app/core/auth/account.service.ts

```diff
@@ -133,10 +136,19 @@ export class AccountService implements IAccountService {
             this.userIdentity = undefined;
         }
 
+        if (this.versionControlAccessTokenRequired === undefined) {
+            this.profileService.getProfileInfo().subscribe((profileInfo) => {
+                this.versionControlAccessTokenRequired = profileInfo.versionControlAccessToken ?? false;
+            });
+        }
+
```

Comment:
Should this also get reset when `force` is set to true?

Reply:
I don't think that's necessary. If a token is needed or not can't be changed without a server reboot anyway as far as I know. The first time the `identity` method is called, the boolean should be set to the right value.